src/i18n/Localizable.xcstrings

```diff
@@ -21818,6 +21930,7 @@
       }
     },
     "Show_Hidden_Files" : {
+      "comment" : "is this one used as gerund or noun?",
```

Comment:
It's an action, so I would say a verb

Reply:
deleted comment and used gerund, same reason for "Restart"